packages/back-end/types/organization.d.ts

```diff
@@ -106,6 +106,178 @@ export interface Namespaces {
   status: "active" | "inactive";
 }
 
+export type SupportedCurrencies =
```

Comment:
@jdorn Very cool trick. I tried to do this a few ways, but ultimately, I had to create a new file in the backend/util folder called `currencies.ts` to define the `supportedCurrencies`. I then defined, referenced, and exported the type `SupportedCurrencies` within `organization.d.ts` for use on the frontend. 

Our linter wasn't happy about me defining a const within a `.d.ts` file, and I don't see us using the pattern of defining something like that on the frontend and referencing it on the backend, only the opposite. 

Let me know what you think of this implementation. The changes are all in [this commit](https://github.com/growthbook/growthbook/pull/1275/commits/e2fc0bfe559c9a87cbc6db965cbb8b82810f80b5).

Reply:
There's a new `shared` package that is meant for sharing code/types between the front-end and the back-end.

However in this case, I think there's a simpler solution. The back-end doesn't really need to care about the specific currencies at all.  It can just type the org setting as a `string`.  So you can move the constant/type definition completely to the front-end.